js/analyses/countLinks.js

```diff
@@ -0,0 +1,49 @@
+/** @module analyses/countLinks */
+
+var getAnchors = require( "../stringProcessing/getAnchorsFromText.js" );
+var findKeywordInUrl = require( "../stringProcessing/findKeywordInUrl.js" );
+var getLinkType = require( "../stringProcessing/getLinkType.js" );
+var checkNofollow = require( "../stringProcessing/checkNofollow.js" );
+
+/**
+ * Checks a text for anchors and returns an object with all linktypes found.
+ *
+ * @param {string} text The text to check for anchors.
+ * @param {string} keyword The keyword to use for matching in anchors.
+ * @param {string} url The url of the page.
+ * @returns {object} The object containing all linktypes.
```

Comment:
We should document what the object contains instead of only putting `{object}` here.